bosh_aws_cpi/lib/cloud/aws/stemcell_creator.rb

```diff
@@ -22,7 +22,7 @@ def create(volume, ebs_volume, image_path)
       ResourceWait.for_snapshot(snapshot: snapshot, state: :completed)
 
       params = image_params(snapshot.id)
-      image = region.images.create(params)
+      image = region.images[region.client.register_image(params).image_id]
```

Comment:
I changed this because the [`images.create`](https://github.com/aws/aws-sdk-ruby/blob/v1.60.2/lib/aws/ec2/image_collection.rb#L193) function wants to convert `{'sdb':'ephemeral0'}` Hash values, but we're giving it the full `BlockDeviceMapping` type.